code/modules/zombie/organs.dm

```diff
@@ -9,8 +9,8 @@
 	var/living_transformation_time = 30
 	var/converts_living = FALSE
 
-	var/revive_time_min = 450
-	var/revive_time_max = 700
+	var/revive_time_min = 60 SECONDS
+	var/revive_time_max = 100 SECONDS
```

Comment:
This should be documented I think

Reply:
It literally is and has been.

![Screenshot_20231011-183157](https://github.com/BeeStation/BeeStation-Hornet/assets/9547572/cad5dcec-1ba3-4d09-85c2-aa6965a79dad)

Reply:
I have been informed that the CL is supposed to be extremely detailed and not just a rough summary.

I will do this going forward.

Reply:
The CL should include every change to a reasonable level (Maps don't need 500 CL entries detailing every changes, but there should be some information about what was changed (sec, eng, etc.) and for balance changes it should list the new values)